components/locales/en-US.json

```diff
@@ -103,7 +103,7 @@
       "yearFormat": "yyyy",
       "dateFormat": "M/d/yyyy",
       "dayFormat": "d",
-      "dateTimeFormat": "M/d/yyyy HH:mm:ss",
+      "dateTimeFormat": "M/d/yyyy hh:mm:ss tt",
```

Comment:
`dateTimeFormat` have to be 24-hours, need to add `timeFormat12Hour` and  `dateTimeFormat12Hour`.

Reply:
This should be added to DatePicker/Calendar in every JSON locale file, not only en-US.json? Right?

Reply:
Yes, but it can fallback to en-US using the default values of the Locale Class.

Reply:
Ok. I've added it to the en-US.